### PR TITLE
[chore] added required packages and updated README for clarity

### PR DESCRIPTION
**Pull request by Gidsss**
Use `npm install --legacy-peer-deps` for installation of packages

### CODE REVIEW

**Review by HolyShaq — APPROVED** [review]

**Comment by HolyShaq** [comment]
para san pala ung `--legacy-peer-deps`?

**Comment by dssrts** [comment]
> para san pala ung `--legacy-peer-deps`?

(1)

**Review by dssrts — APPROVED** [review]

**Comment by Gidsss** [comment]
> para san pala ung `--legacy-peer-deps`?

hello @HolyShaq para ignore peer dependency conflicts (hopefully walang conflicts sa version ng packages HAHAHA)

File: README.md [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
-```bash
-run npm install next
+### Installation
+Clone the repository and install dependencies:
 
-then next,
+```bash
+git clone <your-repo-url>
+cd dataviz-tool
+npm install
```
Comment (HolyShaq) [inline README.md:23]: I see, then mas maganda ata if i-change narin to to match

**Comment by HolyShaq** [comment]
I see, then ichange nalang din ung npm install sa readme para safe then merge ko na

**Comment by Gidsss** [comment]
> I see, then ichange nalang din ung npm install sa readme para safe then merge ko na
> 

yeah nalimutan ko rin lagay pala mb mb add ko na lang din then merge na if goods na, thanks! 🚀

**Comment by HolyShaq** [comment]
oki thanks!

**Pull request merged**